chapter-07/recipe-09/fortran-example/tests/CMakeLists.txt

```diff
@@ -5,7 +5,5 @@ add_test(
   NAME
```

Comment:
Does the top-level `CMakeLists.txt` for this recipe need this?
```
set(CMAKE_Fortran_MODULE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/modules)
```
At least just for consistency with what is shown later in the book?